backend/GymBeam/Controllers/AuthenticationController.cs

```diff
@@ -23,11 +25,66 @@ public class AuthenticationController : ControllerBase
     {
         private readonly IConfiguration _configuration;
         private readonly IMediator _mediator;
+        private readonly IRepository _repository;
+        private readonly GoogleClient _googleClient;
 
-        public AuthenticationController(IConfiguration configuration, IMediator mediator)
+        public AuthenticationController(IConfiguration configuration, IMediator mediator, GoogleClient googleClient, IRepository repository)
         {
             _configuration = configuration;
             _mediator = mediator;
+            _googleClient = googleClient;
+            _repository = repository;
+        }
+
+        [AllowAnonymous]
+        [HttpGet("google")]
+        public IActionResult GetGoogleLoginLink()
+        {
+            var loginLink = _configuration.GetGoogleLoginLink();
+            return Ok(new { link = loginLink });
+        }
+
+        [AllowAnonymous]
```

Comment:
It should has possible error codes probably?

Reply:
done